src/utils/utils.ts

```diff
@@ -375,7 +376,7 @@ export const prepareArchiveFromItem = async ({
     zipStream.on('close', () => {
       // set reply headers depending zip file and return file
       const buffer = fs.readFileSync(zipPath);
-      reply.raw.setHeader('Content-Disposition', `filename="${item.name}.zip"`);
+      reply.raw.setHeader('Content-Disposition', `filename="${slugify(item.name)}.zip"`);
```

Comment:
```suggestion
      reply.raw.setHeader('Content-Disposition', `filename="${slugify(item.name, { remove: /["'`]/g })}.zip"`);
```

Reply:
I'll apply your suggested change and merge, it's still better than nothing as we had. I've opened an issue for when we have time to investigate longer.